lib/install/management/store_files.go

```diff
@@ -139,6 +139,12 @@ func (d *Dispatcher) deleteDatastoreFiles(ds *object.Datastore, path string, for
 	return true, nil
 }
 
+func (d *Dispatcher) geVC65() bool {
+	vcVersion := d.session.Client.ServiceContent.About.Version
+	// vsphere only has versions 5.5.x, 6.0.x, 6.5.x, not likely to have 6.10.x, so compare string directly here
+	return strings.Compare(vcVersion, "6.5.0") >= 0
```

Comment:
so you mean anything without "6.0." prefix? In case, vsphere does not provide any patch like 6.1.x. 
anyway, not likely...